var/spack/repos/builtin/packages/gdl/package.py

```diff
@@ -27,7 +27,8 @@ class Gdl(CMakePackage):
     variant('hdf5', default=True, description='Enable HDF5')
     variant('openmp', default=True, description='Enable OpenMP')
     variant('proj', default=True, description='Enable LIBPROJ4')
-    variant('python', default=False, description='Enable Python')
+    variant('embed-python', default=False, description='Ability to embed Python within GDL')
```

Comment:
We might want to call this `embed_python`. It can be kind of confusing when you see a chain of variants like: `-python+embed-python` since `-` can be used to negate a variant.